packages/app/src/app/components/CreateNewSandbox/CreateSandbox/CreateSandbox.tsx

```diff
@@ -89,108 +98,115 @@ export const CreateSandbox: React.FC<CreateSandboxProps> = props => {
-                        />
-                      </svg>
-                    </CloseModal>
-                  ) : null}
-                </MobileTabs>
+        <Panel tab={tab} id="create">
+          {/**
+           * ❗️ TODO: Update MobileTabs (because they aren't anymore) and move close button
+           * higher up in the tree.
+           */}
+          <MobileTabs>
+            {/* ❗️ TODO: Figure out when it isn't a modal */}
+            {isModal ? (
+              <CloseModal
+                type="button"
+                onClick={() => actions.modals.newSandboxModal.close()}
+              >
+                <svg width={10} height={10} fill="none" viewBox="0 0 10 10">
+                  <path
+                    fill="#fff"
+                    d="M10 .91L9.09 0 5 4.09.91 0 0 .91 4.09 5 0 9.09l.91.91L5 5.91 9.09 10l.91-.91L5.91 5 10 .91z"
+                  />
+                </svg>
+              </CloseModal>
+            ) : null}
+          </MobileTabs>
 
-                <Create collectionId={props.collectionId} />
-              </div>
-            )
-          }
-        </TabContent>
-        <TabContent {...tab} stopId="Explore">
-          {rProps =>
-            !rProps.hidden && (
-              <div {...rProps}>
-                <Explore collectionId={props.collectionId} />
-              </div>
-            )
-          }
-        </TabContent>
-        <TabContent {...tab} stopId="Import">
-          {rProps =>
-            !rProps.hidden && (
-              <div {...rProps}>
-                <MobileTabs>
-                  {props.isModal ? (
-                    <CloseModal
-                      type="button"
-                      onClick={() => actions.modals.newSandboxModal.close()}
-                    >
-                      <svg
-                        width={10}
-                        height={10}
-                        fill="none"
-                        viewBox="0 0 10 10"
-                        {...props}
-                      >
-                        <path
-                          fill="#fff"
-                          d="M10 .91L9.09 0 5 4.09.91 0 0 .91 4.09 5 0 9.09l.91.91L5 5.91 9.09 10l.91-.91L5.91 5 10 .91z"
-                        />
-                      </svg>
-                    </CloseModal>
-                  ) : null}
-                </MobileTabs>
-                <Import />
-              </div>
-            )
-          }
-        </TabContent>
+          <Create collectionId={collectionId} />
+        </Panel>
+        <Panel tab={tab} id="import">
+          <MobileTabs>
+            {/* ❗️ TODO: Figure out when it isn't a modal */}
+            {isModal ? (
+              <CloseModal
+                type="button"
+                onClick={() => actions.modals.newSandboxModal.close()}
+              >
+                <svg width={10} height={10} fill="none" viewBox="0 0 10 10">
```

Comment:
I know it's not the purpose of this PR, but it hurts so bad to see this svg duplicated :D

Reply:
I wasn't touching it since the facelift will overhaul the SVG's anyway, but I see what you mean 😬